### PR TITLE
7188 deprecate plain raw

### PR DESCRIPTION
**Pull request by mkaufmaner**
_Thanks for wanting to fix something on Sequelize - we already love you long time! Please delete this text and fill in the template below. If unsure about something, just do as best as you're able._

_If your PR only contains changes to documentation, you may skip the template below._

### Pull Request check-list

_Please make sure to review and check all of these items:_

- [x] Does `npm run test` or `npm run test-DIALECT` pass with this change (including linting)?
- [x] Does your issue contain a link to existing issue (Closes #[issue]) or a description of the issue you are solving?
- [x] Have you added new tests to prevent regressions?
- [x] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?
- [x] Have you added an entry under `Future` in the changelog?

### Description of change

Fixes https://github.com/sequelize/sequelize/issues/7188

### CODE REVIEW

**Comment by mkaufmaner** [comment]
Please label this as WIP

File: changelog.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Future
+- [FIXED] Removed support for plain objects with property `raw` (ie `{raw: 'malicious'}`). [#7188](https://github.com/sequelize/sequelize/issues/7188)
```
Comment (sushantdhiman) [inline changelog.md]: It should be REMOVED

File: changelog.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
 - [FIXED] Removed support where `order` value is string and interpreted as `Sequelize.literal()`. [#6935](https://github.com/sequelize/sequelize/issues/6935)
```
Comment (sushantdhiman) [inline changelog.md]: This one too

**Comment by mkaufmaner** [comment]
Closing this pull request because the raw object will be deprecated in https://github.com/sequelize/sequelize/pull/7454

**Pull request closed without merging**